sinonjs/sinon
# Add a how-to guide for speeding up time with lolex

#### Purpose (TL;DR) - mandatory
This PR adds a how-to guide about current ways to control promises/async functions with lolex (https://github.com/sinonjs/sinon/issues/1898).

In a nutshell, there are a few examples: a regular function, an async function, and a Promise (_walk into a bar_). These things are tested in different ways to demonstrate what we can and cannot control.

 #### How to verify - mandatory
1. Check out this branch
2. Follow the steps described [here](https://github.com/sinonjs/sinon/blob/master/docs/CONTRIBUTING.md) to build the documentation website locally (using jekyll)
3. Navigate to http://localhost:4000/ -> "How to" -> "How to speed up time with lolex"
